undercrawler/middleware/autologin.py

```diff
@@ -36,16 +37,21 @@ class AutologinMiddleware:
     - do not block event loop in login() method (instead, collect
     scheduled requests in a separate queue and make request with scrapy).
```

Comment:
Nice catch, thanks! Fixed, I think now the only restriction is a single authorization domain per spider, and now it should be easier to relax if needed.